### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by Rohitprasad83**
# Product list Page
[Product Page](https://games-cart-preview.netlify.app/pages/products.html)

### CODE REVIEW

**Review by HeyNitin — APPROVED** [review]
Looks good to me

File: pages/products.html [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                    </label>
+                </div>
+            </aside>
+            <main class="products">
+                <div class="card card__shadow">
+                    <span class="card__icon right">
+                        <i class="far fa-heart"></i>
+                    </span>
+                    <span class="card__image">
+                        <img src="../assets/image.jpg" alt="card Image" />
```
Comment (Aditi0505) [inline pages/products.html]: Suggestion: Can use a better alt description. Eg. product card or Mario card. Since it is a given that it is an image.

**Review by Aditi0505 — APPROVED** [review]
Left one suggestion. Rest, LGTM

File: pages/products.html (hunk 1 above)
Comment (Rohitprasad83) [reply pages/products.html]: Thanks, will change it.

**Pull request merged**